superset/assets/javascripts/explore/components/controls/VizTypeControl.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Label, Row, Col, FormControl, Modal } from 'react-bootstrap';
+import visTypes from '../../stores/visTypes';
+import ControlHeader from '../ControlHeader';
+
+const propTypes = {
+  description: PropTypes.string,
```

Comment:
If they aren't defined we don't print them, that's the case for `FilterControl`.